docs/userGuide/components/panel.md

```diff
@@ -0,0 +1,225 @@
+# Panel
+
+#### Panel is a flexible container that support collapse and expand its content. It is expandable by default.
+
+<tip-box border-left-color="#00B0F0">
+  <i style="font-style: normal; font-weight: bold; color: dimgray">Example</i><br>
+  <panel header="Header">
+    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Alias dolores excepturi expedita illum impedit ipsa labore qui veniam. Blanditiis exercitationem id ipsum libero molestiae, necessitatibus unde? Amet fugiat fugit molestias?
+  </panel>
+</tip-box>
+
+<tip-box border-left-color="black">
+<i style="font-style: normal; font-weight: bold; color: dimgray">Markup</i>
+
+```html
+<panel header="Header">
+  ...
+</panel>
+```
+</tip-box>
+<br>
+
+#### With `minimized` attribute, panel is minimized into an inline block element.
+
+<tip-box border-left-color="#00B0F0">
```

Comment:
We should create a .css file and keep this value in one place, rather than duplicate it in all the different places.

Reply:
Would be tricky as `tip-box` all share the same classes, without an id. There isn't a convenient way to assign a separate class or id to blue or black panels without modifying the underlying Vue code.